UnityProject/Assets/Resources/Prefabs/Objects/Closets/Closet_Locker_Warden.prefab

```diff
@@ -301,14 +299,14 @@ MonoBehaviour:
   syncMode: 0
   syncInterval: 0.1
   Armor:
-    Melee: 30
-    Bullet: 50
-    Laser: 50
-    Energy: 100
-    Bomb: 0
+    Melee: 20
+    Bullet: 10
+    Laser: 10
+    Energy: 0
+    Bomb: 10
     Rad: 0
-    Fire: 80
-    Acid: 80
+    Fire: 70
+    Acid: 60
```

Comment:
did you make these adjustments on purpose?

Reply:
Weird, no.